internal/config/config.go

```diff
@@ -232,7 +232,7 @@ func (r *Root) Validate() error {
 
 // Bundle defines the configuration for a OPA Control Plane Bundle.
 type Bundle struct {
-	Name          string        `json:"-" yaml:"-"`
```

Comment:
The biggest question I have, is if omitting the `name` field was *intentional* for some reason? Maybe @koponen might know? 🤔